partsApp/src/shared/leftNavbar/leftNavbar.component.js

```diff
@@ -0,0 +1,24 @@
+// Packages
+import jQuery from 'jquery';
+// Internal resources
+import Template from './leftNavbar.pug';
+
+class Controller {
+  constructor () {}
+
+  click (element) {
```

Comment:
There should be an Angular way to do this. Maybe @jtormey / @plondon can provide some hints.